compiler/damlc/daml-visual/d3.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
```

Comment:
Why do we need to check these files in? Can we download them from somewhere?

Reply:
There is no publicly hosted one (did not find one), we will have to host it ourself (might be easy I am not sure). And this is way people behind a firewall will not be effected.

Reply:
https://d3js.org/ has a download button that gives you a zip file. Does that not contain this file? Firewalls are irrelevant here, this is something that we will download during our build process like a ton of other files. Not something users will download.

Reply:
Ah I see, yes at least d3 can be fetched from public URL. Will this need to be done as a step in bazel ?

Reply:
Yes, you can use `http_archive` for that. There is also a zip file for `d3plus` at https://github.com/alexandersimoes/d3plus/releases which seems to bundle `d3` as well so we can probably just use that for both.

Reply:
Done